src/instance.js

```diff
@@ -223,19 +236,38 @@ fakeModelInstance.prototype.setDataValue = function (key, value) {
     this._values[key] = value;
 };
 
+/**
+ * Set is used to update values on the instance
+ * If set is called with an object, it will loop over the object,
+ * and call set recursively for each key, value pair
+ * Sequelize docs: http://docs.sequelizejs.com/class/lib/model.js~Model.html#instance-method-set
+ * @param {String} key Key yo get the value for
+ * @param {Any} value
+ */
+fakeModelInstance.prototype.set = function (key, value) {
```

Comment:
We also have this feature implemented. It is at line 176.

Reply:
And also this will set the object as key to the instance value if an object is passed in as a key.